semana7/aula26/labenusers/src/App.js

```diff
@@ -29,18 +35,24 @@ export default class App extends React.Component {
     }
   }
 
+  paginaDetalhes = () => {
+    if(this.state.pagina === "lista usuario" || "cadastro") {
+      this.setState({ pagina: "detalhes" })
+    }
+  }
+
 
   render() {
     return (
       <ContainerGeral>
         {this.renderizaPagina()}
-        {this.state.pagina !== "lista usuario" && (
+        {this.state.pagina !== "lista usuario" && "detalhes" && (
```

Comment:
Um pequeno erro de sintaxe aqui! Voce nao esta vendo se this.state.pagina é diferente de "detalhes", só esta checando se "detalhes" é true - e é, porque é uma string cheia.